front/app/component-library/components/Accordion/index.tsx

```diff
@@ -116,10 +116,6 @@ const Accordion = ({
   const [isExpanded, setIsExpanded] = useState(isOpenByDefault);
   const uuid = useInstanceId();
 
-  useEffect(() => {
-    setIsExpanded(isOpenByDefault);
-  }, [isOpenByDefault]);
-
   const handleChange = () => {
```

Comment:
I did start using the CollapseContainer in the Accordion, but it became complicated to handle the edge cases for the Accordion. I'll look into this a bit later if time instead.